draft-packages/tag/KaizenDraft/Tag/Tag.tsx

```diff
@@ -133,16 +134,18 @@ const Tag = (props: TagProps) => {
                 return
             }
           })()}
-        <span
-          className={classNames(styles.textContent, {
+        <Paragraph
```

Comment:
Mmm...I'm a little iffy about this change. If we make any change to Paragraph later, it shouldn't be changing the styles of Tag, and this opens us up for that.

Reply:
That's fair, if we're using the design tokens in the scss then that should be fine. My thinking that brought to this solution was to ensure continuity with text styles with little maintenance - mainly to avoid things being out of sync like here